packages/Icon/src/index.js

```diff
@@ -0,0 +1,38 @@
+export { default as Add } from "./Add";
```

Comment:
the consumer application should be aware to apply tree shaking for this isn't it? if not will bundle all the icons correct?

Reply:
Yes, it's usually handled automatically by webpack/rollup.. if case in their apps don't support tree shaking, they can still use the old import way `import Caution from '@paprika/icon/lib/Caution'` to avoid bundling everything.